cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -141,11 +141,11 @@ def update_from_json(cls, descriptor, jsondict, user, filter_tabs=True):
         return cls.update_from_dict(key_values, descriptor, user)
 
     @classmethod
-    def validate_and_update_from_json(cls, descriptor, jsondict, user, filter_tabs=True):
+    def validate_from_json(cls, descriptor, jsondict, user, filter_tabs=True):
```

Comment:
This name is actually more confusing now. This method isn't just validating, as it actually updates the model, while the name implies that this is a non-destructive operation. I think the old name actually still works with the new behavior, and maybe all it takes is a note in the docstring that this method updates but does not save the model.